prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.html

```diff
@@ -163,9 +163,13 @@
               [class]="row.currentStatusCode | configCode: 'statuses' | case: 'space' : 'kebab'">
 
             <div class="d-flex align-items-center">
-              <span class="mr-1">{{ row.currentStatusCode | configCode: 'statuses' | default }}</span>
+              <span class="mr-1 status"
+                    [class.paper-enrolment-complete-incomplete]="filterPaperEnrolmentStatus(row)">
+                    {{ (filterPaperEnrolmentStatus(row) ? paperEnrolmentStatusMap.get(row.currentStatusCode) : row.currentStatusCode | configCode: 'statuses') | default }}</span>
               <ng-container *ngIf="row.gpid | contains: PAPER_ENROLLEE_GPID_PREFIX; else nonPaperEnrolleeIcons">
-                <mat-icon matTooltip="Paper Submission"
+                <mat-icon class="status"
+                          [class.paper-enrolment-complete-incomplete]="filterPaperEnrolmentStatus(row)"
```

Comment:
As far as i can tell to be in this *ngif you have to be a paper enrolment and those are the only two status so this should just be true.

Reply:
I saw that we can also set the status to locked, declined... etc for paper enrolment in the Adjudicator's Actions